### PR TITLE
Contacts/Events/Downloads updated

### PR DESCRIPTION
**Pull request by pgleeson**
The most pressing updates to the contact, events & downloads pages, better reflecting the current state of the project

### CODE REVIEW

File: downloads.html [hunk 1]
```diff
@@ -76,56 +79,107 @@ <h1>Downloads</h1>
 			    <button type="button" class="close" data-dismiss="alert">&times;</button>
 			    <strong>Warning:</strong> You can't download the whole virtual worm, we're not quite there yet! :)
 			</div>
+            			
+            <!-- Docker ================================================== -->
+			<section id="docker">
```
Comment (gidili) [inline downloads.html:84]: Formatting in this section seems a bit crazy 😜

Comment (pgleeson) [reply downloads.html]: Thanks, have improved formatting...

**Comment by pgleeson** [comment]
Thanks @gidili. Looks good on live now.

**Comment by gidili** [comment]
@pgleeson looking good - thanks for the improvements!

**Pull request merged**